features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Blacklist/List.jsx

```diff
@@ -95,6 +98,77 @@ export default function ListBlacklistThrottlingPolicies() {
         }),
     };
 
+    /**
+ * Mock API call
+ * @returns {Promise}.
+ */
+    function apiCall() {
+        let policyList;
```

Comment:
Why we define this `policyList ` variable outside the promise lambda function? I think it's better to move it inside the lambda function scope and refer it

Reply:
fixed